src/Schema/Events/Search.ts

```diff
@@ -87,3 +87,52 @@ export interface SearchedWithNoResults {
   destination_owner_type?: PageOwnerType
   query: string
 }
+
+/**
+ * A user queries the Artsy Price Database
+ *
+ * This schema describes events sent to Segment from [[searchedPriceDatabase]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "searchedPriceDatabase",
+ *    context_module: "priceDatabaseLanding",
+ *    context_owner_type: "priceDatabase",
+ *    destination_owner_type: "artistAuctionResults",
+ *    destination_owner_id: "4e9746234e77d300010010c6",
+ *    destination_owner_slug: "franz-kline",
+ *    query: "franz kline"
+ *
+ *  }
+ * ```
+ */
+export interface SearchedPriceDatabase {
```

Comment:
The gist of this is that users will be searching artist names in combination with some filter params (size, time period, medium) on a specialized landing page/entry point to the Price Database. My understanding is that artists are always sent to the artist-insights tab on the Artist page upon entering a query, so the context and destination props reflect that.